src/views/Correction.vue

```diff
@@ -114,7 +118,7 @@ export default class Correction extends Mixins(CommonMixin) {
       // set entity type for misc functionality to work
       // do not proceed if this isn't a BC or firm correction
       this.setEntityType(filing.business?.legalType)
-      if (!this.isBenBcCccUlc && !this.isFirm) {
+      if (!this.isBenBcCccUlc && !this.isFirm && !this.isCoop) {
```

Comment:
Please update the comment.

PS I know these are all the entity types we support -- today -- but we might have others in the future, and this code will be safe then.